File: app/src/main/java/com/hbjpro/androidvoiceassistant/Interface/OnSpeechResult.java
package com.hbjpro.androidvoiceassistant.Interface;

import com.hbjpro.androidvoiceassistant.Speech.SpeechResult;

public interface OnSpeechResult {
    void onSpeechSuccess(SpeechResult speechResult);
    void onSpeechError(String errorMsg);
    //void onProcessCommandSuccess(CommandResult commandResult);
    //void onProcessCommandError();
}

File: app/src/main/java/com/hbjpro/androidvoiceassistant/Interface/OnSpeechResult.kt
package com.hbjpro.androidvoiceassistant.Interface

import com.hbjpro.androidvoiceassistant.Speech.SpeechResult

interface OnSpeechResult {
    fun onSpeechSuccess(speechResult: SpeechResult)
    fun onSpeechError(errorMsg: String)
    //void onProcessCommandSuccess(CommandResult commandResult);
    //void onProcessCommandError();
}
